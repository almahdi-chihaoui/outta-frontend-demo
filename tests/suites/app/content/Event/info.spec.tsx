import React from "react";
import renderer from "react-test-renderer";

import Info from "../../../../../src/containers/app/content/Event/info";

describe('Info Component', () => {
  let component;

  beforeEach(() => {
    component = renderer.create(
      <Info></Info>
    );
  });

  it('should render', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});