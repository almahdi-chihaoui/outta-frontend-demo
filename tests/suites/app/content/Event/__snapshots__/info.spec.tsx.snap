// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Component should render 1`] = `
<div
  className="MuiGrid-root makeStyles-info-1 MuiGrid-item"
>
  <div
    className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1"
  />
</div>
`;
